dao/src/main/kotlin/repositories/scannerrun/DaoScannerRunRepository.kt

```diff
@@ -93,8 +94,21 @@ class DaoScannerRunRepository(private val db: Database) : ScannerRunRepository {
             scanners.getOrPut(dao.identifier.mapToModel()) { mutableSetOf() }.add(dao.scannerName)
         }
 
-        scannerRunDao.mapToModel()
-            .copy(provenances = provenanceResolutionResults, scanResults = scanResults, scanners = scanners)
+        val ortRunId = scannerRunDao.scannerJob.ortRun.id.value
+        val scannerIssues = GetIssuesForOrtRunQuery(ortRunId, ScannerRunDao.ISSUE_WORKER_TYPE).execute()
+        val issuesById = mutableMapOf<Identifier, MutableSet<Issue>>()
+        scannerIssues.forEach { issue ->
+            issue.identifier?.let { identifier ->
```

Comment:
Nit: Maybe prefer `also` over `let` as the return value of `let` is not use here.